### ISSUE TITLE
Necessity of Autostart

### ISSUE THREAD

**Issue by johnd2010** [ctu-mrs/mrs_uav_autostart#1]
Hi,

Would it be possible to take the drone off manually, switch to pos-hold, switch to guided/offboard, and then run the MRS code as usual, essentially bypassing the autostart altogether?

**Comment by johnd2010** [comment]
Found an option in the config file. Sorry for the confusion

**Comment by klaxalk** [comment]
Yes, there are multiple ways. If you want to take off by service, you can set `handle_takeoff: false` in the autostart's config. You still need to arm the drone and switch to offboard using a transmitter (service in the simulation), but then the takeoff can be triggered by `uav_manager/takeoff` service. Alternatively, there is the option to takeoff fully-manually using an RC control and switch the system in mid-air. That is call `midair activation` and can be triggered by the `uav_manager/midair_activation` service. Just beware, when you do that, the system will go into the offboard mode automatically. So it is a good practice to pre-set the offboard switch into the "on" position before you activate that service.

**Comment by johnd2010** [comment]
Thank you, Tomas, for the detailed response. 

We tried to do a midair activation, but the pixhawk refused to switch to offboard. Is there any particular px4 param that must be set before the midair activation?

In addition to this, we also tried to take off directly with the autostart, however, the motor-thrust behavior was concerning. We have applied the proper motor-thrust values as per the MRS documentation. Since we are not using a rangefinder, I was wondering if the behavior seen is because of that. (We are using GPS-baro)
https://github.com/user-attachments/assets/eec0fdc2-b663-48a8-9dbf-79f6e9076542

The [log](https://drive.google.com/drive/folders/1nGns4tBBWTW7FV5b54aPlwPy_dMc4zCe?usp=sharing) analysis did not raise any concerning errors either. 

PS: We are really excited to work with your vast framework. Thank you for making it open source and for all the support to the framework.

**Comment by klaxalk** [comment]
@johnd2010 thanks for providing all the data! Yes the video suggest there is something wrong. I will check that soon. I would recommend to wait till I get back to you.

Regarding the midair activation: No, there is nothing to be set on the PX4 side. From the PX4 Point-of-view, nothing special happens. Do you have the "tmux logs" from that flight as well?

**Comment by johnd2010** [comment]
Thank you Tomas,

The logs have been uploaded here
 https://drive.google.com/drive/folders/1nGns4tBBWTW7FV5b54aPlwPy_dMc4zCe

**Comment by johnd2010** [comment]
Hi Tomas,

We had the opportunity to try out one more time and we increased the param value max_position_difference of the land_off_tracker from 0.7 to 1.0 

Upon this change, the take off was smooth. Can you please let us know if this value is reasonable or if there is a safer way to achieve this by tuning a different param ? 


In the context of mid air Activation, is there a default waiting period for the control switch from pos_hold to offboard? 

Thanks again, for all the support
